CHANGELOG.md

```diff
@@ -4,6 +4,21 @@
 
 ### Breaking Changes
 - Removed Retesteth rpc service and commands [#7833](https://github.com/hyperledger/besu/pull/7783)
+- With the upgrade of the Prometheus Java Metrics library, there are the following changes:
```

Comment:
there are some projects (eg teku) that use metrics core modules - need to communicate breaking changes to them

Reply:
yes, will support them for the upgrade